src/main.ts

```diff
@@ -1,39 +1,80 @@
 import * as core from '@actions/core';
 import * as fs from 'fs';
+import path from 'path';
 
 import {getAngularVersions} from './get-angular-versions';
 import {overrideAngularVersions} from './override-angular-versions';
+import {replaceLibrariesBuildBuilder} from './replace-libraries-build-builder';
+import {AngularJson} from './types/angular-json';
 import {PackageJsonVersion} from './types/package-json-version';
 
+function ensureCorrectLibraryBuildBuilder(
```

Comment:
suggestion: Rename to `ensureCorrectLibraryBuilder` or `ensureCorrectNgPackagrBuilder`. A *library build builder* sounds a little confusing 😊 Similar names are present other places in this PR.